enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -382,8 +371,27 @@ func (c *ociContainer) createRootfs(ctx context.Context) error {
 
 	// Create an overlayfs with the pulled image layers.
 	var lowerDirs []string
-	for _, d := range c.layerDigests {
-		lowerDirs = append(lowerDirs, layerPath(c.layersRoot, d))
+	layers, ok := c.imageStore.CachedLayers(c.imageRef)
+	if !ok {
+		return fmt.Errorf("bad state: attempted to create rootfs before pulling image")
```

Comment:
Change to InternalError?

Reply:
I think it makes sense to have this be a retryable error for now - in the places where we're calling this func, we're always wrapping with `UnavailableError` currently.